8bitbench/build/rust/pkg/emu_bench.js

```diff
@@ -1,9 +1,11 @@
+let wasm_bindgen;
+(function() {
+    const __exports = {};
+    let wasm;
```

Comment:
Why wrap the body of this script in a function? I assume because this script clobbers the global namespace in a way that breaks another script? Maybe we should have the driver understand the .mjs file hack?

Either way, I'd I think we should keep the indentation consistent. Preferably, we would fix the rest of the file but I'm ok with just not indenting this function.

Reply:
That is actually the output of the Rust build step (`wasm-pack build --target no-modules --release` specifically), so an autogenerated file. I would thus not manually modify/re-indent it and hence placed it in the `build/` directory.